node/src/main/resources/reference.conf

```diff
@@ -78,6 +78,9 @@ rnode {
 
     # Port used for external gRPC API
     port-internal = 40402
+
+    # Maximum size of message that can be sent via gRPC API
+    max-message-size = 256K
```

Comment:
Do you not need to delete the max-message-size above (on line 49)?

Reply:
Ah ok makes sense. In that case can you make the default for API messages 10x the current value?